src/components/bcros/alert/index.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div :data-cy="`alert-display${index || index===0 ? '-' + index : ''}`" class="px-3 py-3">
+    <UIcon
+      v-if="showHeader"
+      :class="`${iconColour} mr-2 font-semibold`"
+      :name="iconName"
+      data-cy="alert-icon"
+    />
+    <span v-if="showHeader" class="font-semibold flex-auto">{{ alertHeader }}</span>
+    <button
+      v-if="showHeader"
+      class="font-semibold float-right text-primary-500 hover:bg-blue-100 text-sm px-1 py-1 mr-4"
+      href="#"
+      @click="expanded = !expanded"
+    >
+      {{ !expanded ? 'View Details' : 'Hide Details' }}
+      <UIcon
+        v-if="showHeader"
+        class="font-semibold text-primary-500 mr-2"
+        :name="!expanded ? 'i-mdi-chevron-down' : 'i-mdi-chevron-up'"
+        data-cy="expand-icon"
+      />
+    </button>
+    <div v-if="expanded && showDescription" :data-cy="`alert-description${index || index===0 ? '-' + index : ''}`">
```

Comment:
I think you can do away with the `${index || index===0 ? '-' + index : ''}` logic here and above and just reference the index you want in your tests right?

i.e. '<selector>.eq(index)' inside the tests and then you don't need the index at all?

example of changes to test:
```
cy.get('[data-cy="alerts-display"]').should('exist')
cy.get('[data-cy="alerts-display"]').should('have.length', 2)
cy.get('[data-cy="alert-display"]').eq(0).should('contain.text', 'This business is frozen')
...
```

Reply:
Done